### DISCUSSION TITLE
Attenuation Tomography

### DISCUSSION THREAD

**Discussion by makeabhishek** [gimli-org/pyGIMLi#736] [Q&A]
I was reading the issue #311 on Attenuation tomography. I'm wondering, what is the input and output of Attenuation tomography? How to obtain the amplitude values for inversion manager?
Let's assume if I have the seismic traces and I can obtain the P-wave traveltime for each source-receiver pair. How can I obtain the amplitude?

Is there any reference available to read about it? 
I was reading about attenuation tomography by Ziqi et al., where they talked about 'Q' factor.  Is that the concept implemented in PyGimli?

Jin, Ziqi, and Zandong Sun. "Attenuated traveltime tomography method for estimation of seismic attenuation." Journal of Applied Geophysics 139 (2017): 73-78.

**Comment by halbmy** [comment]
pyGIMLi is not "formally supporting attenuation tomography" or any specific method like Q-factor, but it should be rather considered a toolbox that can be used to do various things, one of them being traveltime AND attenuation tomography. If you follow the idea of #311, this is a super simple task as the problem is linear (once the traveltime problem is solved). Similar could be possible for the mentioned method.

At any rate, it would be interesting to see an example about attenuation tomography.

What you need is the relative amplitude (measured amplitude divided by source or near source amplitude), it could be added as another data field (column) in the data container (file). How you estimate this amplitude is beyond pyGIMLi, but there are other packages like obspy to analyse seismic traces.

**Reply by makeabhishek** [reply]
Thanks @halbmy for the explanation. I want to clarify the amplitude aspect again. Let's take the acoustic signal from source to receiver. The traveltime is marked as red line in the time trace. As you mentioned above _"measured amplitude divided by source or near source amplitude"_. What is the **'measured amplitude'**.  Does it refers to the amplitude at that traveltime (as shown by red line) normalized by source signal amplitude?   

<img src="https://github.com/gimli-org/gimli/assets/47937684/aa73647b-e88f-443c-9c43-d6d6dcea561d" width=35% height=35%>

**Reply by halbmy** [reply]
I guess we will need experts in this field, I am none.

**Reply by makeabhishek** [reply]
I read few articles and found that there are different ways to obtain amplitude. For example by frequency shift method, ratio of magnitude in frequency domain, etc.
